src/test/java/outcomes/process_executor/go/find_more_then_one_executable_file/FindMoreThenOneExecutableFile.java

```diff
@@ -0,0 +1,30 @@
+package outcomes.process_executor.go.find_more_then_one_executable_file;
+
+import org.hyperskill.hstest.dynamic.DynamicTest;
+import org.hyperskill.hstest.testcase.CheckResult;
+import org.hyperskill.hstest.testing.TestedProgram;
+import outcomes.base.ContainsMessage;
+import outcomes.base.UserErrorTest;
+
+
+public class FindMoreThenOneExecutableFile extends UserErrorTest<String> {
+
+    @ContainsMessage
+    String m1 = "Cannot decide which file to run out of the following: ";
+
+    @ContainsMessage
+    String m2 = "main1.go\"";
+
+    @ContainsMessage
+    String m3 = "main2.go\"";
+
+    @ContainsMessage
```

Comment:
Please, also add 

```java
@NotContainMessage
String m5 = "main.go";
```

We should be sure that `hs-test` is not considering this file because it's not in the directory.